backend/src/main/java/de/bund/digitalservice/ris/norms/config/PortalPrototypeBucketServiceConfiguration.java

```diff
@@ -22,7 +22,7 @@
  * </ul>
  */
 @Configuration
-@Profile({ "production", "local", "local-docker" })
+@ConditionalOnProperty("publish.portal-prototype.enabled")
```

Comment:
Oh wait now we also don't need this in local and local-docker? I forgot what was the reason..